validator/templates/validator/datasets.html

```diff
@@ -29,6 +30,7 @@
             </td>
 			<td>The QA4SM tool supports version v201706 and v201812 of the C3S product. Currently, the COMBINED product (combination
                 of active and passive sensors) is supported.</td>
+            <td>1978 - 2018 (both versions)</td>
 			<td>The data can be downloaded from the <a target="_blank" href="https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-soil-moisture?tab=overview">Climate Data Store</a>.</td>
 			<td>TBA</td>
```

Comment:
https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-soil-moisture?tab=overview
Use this link for the citation for now

Reply:
updated accordingly